app/client/components/payment/update/paymentUpdated.tsx

```diff
@@ -48,11 +49,16 @@ const ConfirmedNewPaymentDetailsRenderer = (subscription: Subscription) => {
               </div>
             ) : (
               <>
-                <div>
-                  <b>Next Payment:</b> {subscription.plan.currency}
-                  {(subscription.nextPaymentPrice / 100.0).toFixed(2)} on{" "}
-                  {formatDate(subscription.nextPaymentDate)}
-                </div>
+                {subscription.nextPaymentPrice &&
```

Comment:
these can be null for print products, so only shows this on the payment confirmed page if it has a `nextPaymentPrice` and `nextPaymentDate`